examples/tls_rustls_dynamic_certs.rs

```diff
@@ -176,18 +173,15 @@ impl ResolvesServerCert for DynamicIssuer {
         let client_hello = ClientHello::rama_from(client_hello);
 
         let key = match client_hello.ext_server_name() {
-            Some(host) => match host {
-                rama::net::address::Host::Name(domain) => {
-                    if domain == &Domain::from_static("example") {
-                        self.example_data.clone()
-                    } else if domain == &Domain::from_static("second.example") {
-                        self.second_example_data.clone()
-                    } else {
-                        self.example_data.clone()
-                    }
+            Some(domain) => {
+                if domain == &Domain::from_static("example") {
```

Comment:
https://ramaproxy.org/docs/rama/net/address/struct.Domain.html#impl-PartialEq%3C%26str%3E-for-Domain same here